Source/ApiGWs/Tailwind.Traders.WebBff/Controllers/ProfilesController.cs

```diff
@@ -34,12 +35,13 @@ public class ProfilesController : Controller
         [ProducesResponseType((int)HttpStatusCode.OK)]
         public async Task<IActionResult> GetProfiles()
         {
+
             var client = _httpClientFactory.CreateClient(HttpClients.ApiGW);
 
             var result = await client.GetStringAsync(API.Profiles.GetProfiles(_settings.ProfileApiUrl, VERSION_API));
             var profiles = JsonConvert.DeserializeObject<IEnumerable<Profile>>(result);
 
-            return Ok(profiles);
```

Comment:
Hi @cdemiguel 
Why is Ok(profiles) changed to Ok()? We don't need to return the profiles from this endpoint? If not, what is the reason of it?

Reply:
i was doing some test, in one of them i deleted it,
now is fixed,
thanks, @eiximenis,